nilearn/decoding/tests/test_decoder.py

```diff
@@ -94,50 +95,53 @@ def test_BaseDecoder_check_estimator():
     # if it is a string, and if not, then raise the error
     supported_estimators = ['svc', 'svc_l2', 'svc_l1',
                             'logistic', 'logistic_l1', 'logistic_l2',
-                            'ridge', 'ridge_classifier', 'ridge_regressor',
-                            'svr']
+                            'ridge', 'ridge_classifier',
+                            'ridge_regressor', 'svr']
     unsupported_estimators = ['ridgo', 'svb']
     expected_warnings = ['Use a custom estimator at your own risk '
                          'of the process not working as intended.']
 
     with warnings.catch_warnings(record=True) as raised_warnings:
         for estimator in supported_estimators:
-            BaseDecoder(estimator=estimator)._check_estimator()
+            _BaseDecoder(estimator=estimator)._check_estimator()
     warning_messages = [str(warning.message) for warning in raised_warnings]
     for expected_warning_ in expected_warnings:
         assert expected_warning_ not in warning_messages
     
     for estimator in unsupported_estimators:
         assert_raises(ValueError, 
-                      BaseDecoder(estimator=estimator)._check_estimator)
+                      _BaseDecoder(estimator=estimator)._check_estimator)
 
     custom_estimator = random_forest
     assert_warns(UserWarning,
-                 BaseDecoder(estimator=custom_estimator)._check_estimator)
+                 _BaseDecoder(estimator=custom_estimator)._check_estimator)
 
 
 def test_parallel_fit():
     # The goal of this test is to check that results of _parallel_fit is the
     # same for differnet controlled param_grid
+    X, y = make_regression(n_samples=100, n_features=20,
+                           n_informative=5, noise=0.2, random_state=42)
     train = range(80)
-    test = range(80, len(y_regression))
+    test = range(80, len(y_classif))
     outputs = []
-    estimator = ridge
+    estimator = svr
+    # two param lists for svr
+    list_params = [[1e-1, 1e0, 1e1], [1e-1, 1e0, 5e0, 1e1]]
     # define a scorer
     scorer = check_scoring(estimator, 'r2')
-    # check two params lists for ridge
-    for params in [[1e-1, 1, 10], [10, 1e-1, 0, 1]]:
+
+    for params in list_params:
         param_grid = {}
-        param_grid['alpha'] = np.array(params)
+        param_grid['C'] = np.array(params)
```

Comment:
`C` is not a good name. Writing the full name here would make the test easier to comprehend.

Reply:
I see that `C` has been used elsewhere as well, with `svr`. Is `C` a standard pre-existing terminology?

Reply:
yes, and it is also the name for the svr scikit-learn estimator hyperparameter